bundler/spec/support/helpers.rb

```diff
@@ -244,10 +244,9 @@ def create_file(path, contents = "")
       if contents.start_with?("#!")
         path.chmod(0o755)
         if Gem.win_platform?
-          bat_path = path.dirname.join(path.basename.to_s + ".bat")
-          bat_path.write(strip_whitespace(<<-SCRIPT))
-            @ECHO OFF
-            @"ruby.exe" "%~dpn0" %*
+          path.sub_ext(".bat").write <<~SCRIPT
+@ECHO OFF
+@"ruby.exe" "%~dpn0" %*
```

Comment:
Does it not work with "nice" indentation? The `<<~` is supposed to strip whitespace so you can keep your heredocs looking pretty.

Reply:
oh I don't know that ( this is a habit brought from python)